test/e2e/e2ethanos/services.go

```diff
@@ -1026,3 +1026,66 @@ func NewS3Config(bucket, endpoint, basePath string) s3.Config {
 		},
 	}
 }
+
+// NOTE: by using aggregation all results are now unsorted.
+var QueryUpWithoutInstance = func() string { return "sum(up) without (instance)" }
+
+// DefaultPromConfig returns Prometheus config that sets Prometheus to:
+// * expose 2 external labels, source and replica.
+// * optionallly scrape self. This will produce up == 0 metric which we can assert on.
+// * optionally remote write endpoint to write into.
+func DefaultPromConfig(name string, replica int, remoteWriteEndpoint, ruleFile string, scrapeSelf bool, scrapeTargets ...string) string {
```

Comment:
Maybe we could play with something like 

`DefaultPromConfig(name string, replica int, remoteWriteEndpoint, ruleFile string, scrapeTargets ...string)` and have constant for `const LocalPrometheusScrape string = "localhost:9090"`? Not a biggie (:

Reply:
I like this idea much more :+1: Better than adding new parameter. Changed.